src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -92,67 +57,109 @@ export default {
 
       this.node.definition.targetRef = targetShape.component.node.definition;
       this.sourceShape.component.node.definition.get('outgoing').push(this.node.definition);
-      targetShape.component.node.definition.get('incoming').push(this.node.definition)
+      targetShape.component.node.definition.get('incoming').push(this.node.definition);
 
       this.updateWaypoints();
 
       targetShape.attr({
-        body: { fill: '#fff', cursor: 'move' },
+        body: { fill: `${this.defaultNodeColor}`, cursor: 'move' },
         label: { cursor: 'move' },
+
+        ".body": { fill: `${this.defaultNodeColor}`, cursor: 'move' },
+        ".label": { cursor: 'move' },
       });
 
       this.shape.listenTo(this.sourceShape, 'change:position', this.updateWaypoints);
       this.shape.listenTo(targetShape, 'change:position', this.updateWaypoints);
     },
+    checkExclusiveGateway() {
+      const definition = this.target.component.node.definition;
+      const gatewayDirection = definition.get('gatewayDirection');
+      const incoming = definition.get('incoming');
+      const outgoing = definition.get('outgoing');
+      const incomingFlowCount = incoming.length;
```

Comment:
Since some variables are only used once, and `get('incoming')` and `get('outgoing')` will always return an array, you can cut down on intermediate variables and this may help with readability.

Also, `Diverging` seems to be the default option, so we can further clean this up by placing the non-default flow first and using an early return.

```
const definition = this.target.component.node.definition;
const gatewayDirection = definition.get('gatewayDirection');
const incomingFlowCount = definition.get('incoming').length;
const outgoingFlowCount = definition.get('outgoing').length;

if (gatewayDirection == this.gatewayDirectionOptions.Converging) {
  return true;
}

if (incomingFlowCount === 0) {
  return true;
}

if (outgoingFlowCount > 0) {
  return true;
}

return false;
```

If there were more than two options, we can worry about being more explicit, and possibly use a `switch` statement, but I think this is nice considering the two options right now.